### DISCUSSION TITLE
Import existing metadata from excel/csv format

### DISCUSSION THREAD

**Discussion by BobbyAxelrods** [open-metadata/OpenMetadata#9699] [Ideas]
I would like to suggest if we can have an import feature where one can upload existing metadata from an excel instead of to key in the description 1 by 1. If 10 columns then should be ok, buat what if one have 100 columns and need to fill the description 1 by 1 manually. 

Thanks.

**Comment by pmbrull** [comment]
Hi, you should be able to handle this programmatically with the APIs. You can find here some more information on how to use the Python SDK https://docs.open-metadata.org/sdk/python
